common/ethnicities/wc_ethnicities_elven.txt

```diff
@@ -248,30 +143,70 @@ high_elven = {
         10 = {  name = complexion_4             range = { 0.75 1.0 }     }
         10 = {  name = complexion_5             range = { 0.75 1.0 }     }
 	}
+	#Height
+	gene_height = {
+        1 = { name = normal_height  range = { 0.5 0.55 }      }
+    }
 	#Noses
+    gene_bs_nose_forward = {
+        1 = { name = nose_forward_pos    range = { 0.6 1.0   }    }
+    }
 	gene_bs_nose_height = {
-        1 = { name = nose_height_neg    range = { 0.25 0.75 }    }
+        1 = { name = nose_height_neg    range = { 0.4 0.8 }    }
     }
 	gene_bs_nose_nostril_height = {
-        1 = { name = nose_nostril_height_neg    range = { 0.25 0.75 } }
+        1 = { name = nose_nostril_height_neg    range = { 0.2 0.5 } }
     }
 	gene_bs_nose_nostril_width = {
         1 = { name = nose_nostril_width_neg    range = { 0 0.5 }    }
     }
 	gene_bs_nose_profile = {
-        1 = { name = nose_profile_neg    range = { 0.25 0.75   }    }
+        1 = { name = nose_profile_neg    range = { 0.1 0.3   }    }
+    }
+    gene_bs_nose_ridge_angle = {
+        1 = { name = nose_ridge_angle_pos    range = { 0.0 0.6   }    }
+    }
+    gene_bs_nose_ridge_width = {
+        1 = { name = nose_ridge_width_neg    range = { 0.0 0.0 }    }
+    }
+    gene_bs_nose_size = {
+        1 = { name = nose_size_neg    range = { 0.0 0.1 }    }
+		1 = { name = nose_size_pos    range = { 0.0 0.1 }    }
     }
 	gene_bs_nose_tip_angle = {
-		1 = { name = nose_tip_angle_neg   range = { 0 0.2 } }
-        1 = { name = nose_tip_angle_pos   range = { 0 0.3 } }
+		1 = { name = nose_tip_angle_neg   range = { 0.0 0.2 } }
+        1 = { name = nose_tip_angle_pos   range = { 0 0.2 } }
     }
-	gene_bs_nose_length = {
-        1 = { name = nose_length_neg    range = { 0.5 1 }    }
+    gene_bs_nose_tip_forward = {
+        1 = { name = nose_tip_forward_neg    range = { 0.4 0.0 }    }
```

Comment:
A range must be from the lowest value to the highest. Otherwise, it's reported as an error.
Here's a list of them.
```
[22:27:55][ethnicity.cpp:97]: invalid range at  file: common/ethnicities/wc_ethnicities_elven.txt line: 181
[22:27:55][ethnicity.cpp:97]: invalid range at  file: common/ethnicities/wc_ethnicities_elven.txt line: 184
[22:27:55][ethnicity.cpp:97]: invalid range at  file: common/ethnicities/wc_ethnicities_elven.txt line: 187
[22:27:55][ethnicity.cpp:97]: invalid range at  file: common/ethnicities/wc_ethnicities_elven.txt line: 276
[22:27:55][ethnicity.cpp:97]: invalid range at  file: common/ethnicities/wc_ethnicities_elven.txt line: 279
[22:27:55][ethnicity.cpp:97]: invalid range at  file: common/ethnicities/wc_ethnicities_elven.txt line: 288
[22:27:55][ethnicity.cpp:97]: invalid range at  file: common/ethnicities/wc_ethnicities_elven.txt line: 391
[22:27:55][ethnicity.cpp:97]: invalid range at  file: common/ethnicities/wc_ethnicities_elven.txt line: 409
[22:27:55][ethnicity.cpp:97]: invalid range at  file: common/ethnicities/wc_ethnicities_elven.txt line: 419